themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1365,7 +1365,7 @@ html body.tc-body.tc-single-tiddler-window {
 	margin-top: 8px;
 }
 
-.tc-tiddler-edit-frame .tc-dropzone-editor > div > div > .tc-editor-toolbar {
+.tc-tiddler-frame .tc-grid.tc-tiddler-preview .tc-editor-toolbar, .tc-tiddler-frame .tc-grid.tc-no-tiddler-preview .tc-editor-toolbar {
```

Comment:
please use 2 lines for better maintainability.

Reply:
I cleaned that up